frontend/src/app/register/register.component.html

```diff
@@ -0,0 +1,50 @@
+<h2>Rejestracja w Volontulo</h2>
+<form method="post" role="form" (ngSubmit)=register()>
+  <div class="form-group">
+    <p>Email:</p>
+    <input type="email" [(ngModel)]="registerModel.email" name="email" class="form-control" required email="true" #email="ngModel"
```

Comment:
You can use just `email` instead `email="true"`. What is more important this validator isn't good enough, because it asses `example@email` as valid, so try to use pattern, see https://github.com/CodeForPoznan/volontulo/blob/2b1069b6080c2997092da5fa93525435f6ed1aaf/frontend/src/app/password-reset/password-reset.component.html#L21
Additionally, you can set `maxlength=150` as it is set like this in user serializer.